unabridged-works-volume-1/original/08-to-the-parlaiment.adoc

```diff
@@ -4,8 +4,8 @@
 [.section-date]
 1659
 
-THAT there hath been a backsliding and turning aside from
-the GOOD OLD CAUSE even by the army (who formerly were glorious
+That there hath been a backsliding and turning aside from
+the _Good Old Cause_ even by the army (who formerly were glorious
```

Comment:
This was uncapitalized and italicized further down in the paragraph.

Reply:
I like how you did it here, let's go with it. 👍